lib/__tests__/integration.test.mjs

```diff
@@ -7,7 +7,7 @@ import sassSyntax from 'postcss-sass';
 import scssSyntax from 'postcss-scss';
 import { stripIndent } from 'common-tags';
 
-import stylelint from '../index.js';
+import stylelint from 'stylelint';
```

Comment:
This uses https://github.com/stylelint/stylelint/pull/7159/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R124, which is a commonjs file, correct?

Reply:
Good catch! I missed `lib/index.cjs` (commonjs) was used.
Fixed via 3bf3fc030. 👍🏼